### DISCUSSION TITLE
Moe's BHT-002-GCLZB target temperature spikes

### DISCUSSION THREAD

**Discussion by Cage82** [Koenkk/zigbee2mqtt#19219] [Devices]
Hello!

So lately I've started experiencing random spikes in Moe's BHT-002-GCLZB target temperatures.

I have around 10 of these thermostats around the house in different rooms. I am using Z2M with Zig-star v4 usb dongle. Lately nothing has changed in my setup except I added a shielded usb cable between the dongle and the computer. So the dongle location has changed maybe 30cm from where it was earlier. I will fiddle around that to check if there is any effect.

But I find it strange that the device changes the target temperatures itself for a second, then changes back to the old value again. Momentarily I've had situation where my garage temperature rose by ~6 Celcius degrees when the target value randomly changed in a rapid manner. There's also cases when the target value had dropped to 0 for several minutes / an hour.

Any idea what could cause this / what should I check to get hang of it / are there any settings I should add in z2m? Catching these events in the logs is quite difficult since the Moe's are sending so much messages around... 

I've set the debounce option in my z2m configuration.yaml followingly:
`   
    friendly_name: TH Moes Termostaatti
    debounce: 1
`

My Z2M settings at the moment:
`
    "child_lock": "UNLOCK",
    "current_heating_setpoint": 18,
    "deadzone_temperature": 1,
    "heat": "OFF",
    "linkquality": 83,
    "local_temperature": 20.5,
    "local_temperature_calibration": -3,
    "max_temperature": 35,
    "max_temperature_limit": 24,
    "min_temperature_limit": 15,
    "preset": "hold",
    "preset_mode": "hold",
    "running_state": "idle",
    "sensor": "AL",
    "system_mode": "heat"
`

A sample picture below, for a moment I had target temperature of 65 thousand Celcius degrees there...
![image](https://github.com/Koenkk/zigbee2mqtt/assets/116311843/a36313f7-caab-4cf3-af26-66bfdc918fbf)

**Comment by RichRandall** [comment]
I have the same issue with this device, which I installed a couple of weeks ago. I have a HA automation that notifies me when it happens (several times per day), but also haven't been able to grab a log. As far as I can tell it's only momentary and doesn't cause the heating to actually activate.

It makes any graphs unreadable because the scale required to fit the spurious values makes any real data invisible.

**Comment by 4ntsu** [comment]
I'm experiencing the same on a daily basis. As mentioned above, it doesn't seem to actually trigger the boiler, but it's annoying having unreadable graphs.
![Screenshot_20240510-200732](https://github.com/Koenkk/zigbee2mqtt/assets/46565592/7c97e79d-adc6-4e15-8b99-f62a616ecce7)
![Screenshot_20240510-201007~2](https://github.com/Koenkk/zigbee2mqtt/assets/46565592/ee6c0e82-944c-453f-8748-16343a8eda4c)
